olive/engine/engine.py

```diff
@@ -376,7 +376,7 @@ def setup_passes(self, accelerator_spec: "AcceleratorSpec"):
         # clean the passes
         self.passes.clear()
         for name, config in self.pass_config.items():
-            pass_cls: Type["Pass"] = config["type"]
```

Comment:
will fail

Reply:
https://stackoverflow.com/questions/77661139/how-do-i-stop-the-ruff-linter-from-moving-imports-into-a-type-checking-if-block